src/main/java/service/UserServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package service;
+
+import lombok.SneakyThrows;
+import repository.UserRepository;
+import dto.UserDto;
+import entity.User;
+import mapper.Mapper;
+import mapper.UserMapper;
+
+import java.util.List;
+
+public class UserServiceImpl implements UserService {
+    private final UserRepository userRepository;
+    private final Mapper<User, UserDto> mapper = new UserMapper();
+
+    public UserServiceImpl() {
+        this.userRepository = new UserRepository();
+    }
+
+    @Override
+    public List<UserDto> getAllUsers() {
+        List<User> userList = userRepository.getAllUsers();
+        return userList.stream().map(mapper::toDto).toList();

```

Comment:
Обычно запись stream происходит в таком стиле:
```suggestion
        return userList.stream()
                        .map(mapper::toDto)
                        .toList();
```

Reply:
исправил